makefile.uni.py

```diff
@@ -148,6 +149,8 @@ def gen_userfile_content(project):
                                                 .format(config['override_build_version']))
 
         if config['Appveyor_Build']:
+            if restore_nuget:
+                raise Exception("Unable to restore Nuget packages with a JOM Makefile build. Figure out how to use MSBuild here.")
```

Comment:
I think we can just use the MSBuild workflow here. JOM allows us to get a progress bar but was also a workaround for a time when there were issues using MSBuild during the transition to VS 2019 and potential issues with utilizing all CPU cores.

At this point those issues are resolved, I believe.